composer.json

```diff
@@ -22,7 +22,9 @@
         "json-ld/tests": "@dev"
     },
     "autoload": {
-        "psr-0": { "ML\\JsonLD": "" }
+        "psr-4": { "ML\\JsonLD\\": "src/" }
```

Comment:
It would be way easier to just change the line above to `"psr-4": { "ML\\JsonLD\\": "" }` and leave the rest as is. I'm not a big fan of hiding everything in a `src` directory. Also, please make sure the tests pass.